arm-documentdb/2015-04-08/documentdb.json

```diff
@@ -0,0 +1,767 @@
+        }
+      },
+      "required": [
+        "tags"
+      ]
+    },
+    "DatabaseAccountListReadOnlyKeysResult": {
+      "description": "The read-only access keys for the given database account.",
+      "properties": {
+        "primaryReadonlyMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the primary read-only key."
+        },
+        "secondaryReadonlyMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the secondary read-only key."
+        }
+      }
+    },
+    "DatabaseAccountListKeysResult": {
+      "description": "The access keys for the given database account.",
+      "properties": {
+        "primaryMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the primary read-write key."
+        },
+        "secondaryMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the secondary read-write key."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DatabaseAccountListReadOnlyKeysResult"
+        }
+      }
+    },
+    "DatabaseAccountRegenerateKeyParameters": {
+      "type": "object",
+      "description": "Parameters to regenerate the keys within the database account.",
+      "properties": {
+        "keyKind": {
+          "type": "string",
+          "description": "The access key to regenerate.",
+          "enum": [
+            "primary",
+            "secondary",
+            "primaryReadonly",
+            "secondaryReadonly"
+          ],
+          "x-ms-enum": {
+            "name": "KeyKind",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "DatabaseAccountOfferType": {
+      "description": "The offer type for the DocumentDB database account.",
+      "type": "string",
+      "enum": [
+        "Standard"
+      ]
+    },
+    "Tags": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      },
+      "description": "Tags are a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters."
+    }
+  },
+  "parameters": {
+    "subscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "description": "Azure Subscription ID.",
+      "required": true,
+      "type": "string"
+    },
+    "apiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Version of the API to be used with the client request. Current version is 2016-04-02"
```

Comment:
`"version": "2015-04-08"` However the description says 2016-04-02. Please provide one value. We have a swagger spec per api-version.